web/src/ApplicationEditPage.js

```diff
@@ -653,8 +654,11 @@ class ApplicationEditPage extends React.Component {
               <div style={{width: "900px", height: "300px"}} >
                 <CodeMirror
                   value={this.state.application.signinHtml}
-                  options={{mode: "htmlmixed", theme: "material-darker"}}
-                  onBeforeChange={(editor, data, value) => {
+                  height="100%"
```

Comment:
The color is not the same as before

![image](https://github.com/user-attachments/assets/d5d16b45-7e04-4295-9188-dba36ee06137)